addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -1140,9 +1140,11 @@ var utils = Object.assign({
             ['res_model', '=', 'ir.ui.view'],
             '|',
             '|',
-            ['name', '=like', '%.assets\_%.css'],
-            ['name', '=like', '%.assets\_%.js'],
+            '|',
+            ['name', '=like', '%.%assets\_%.css'],
```

Comment:
Why this change ?

Reply:
If we do not add this change we will get some system generated asset files in document tab of media dialog. And if user deletes that files thinking that it is unnecessary files then it can lead to a problem. You can see the image below for assest files
![Screenshot from 2022-03-03 21-59-29](https://user-images.githubusercontent.com/72728567/156608960-8a881070-a516-475a-bc3d-cd012dcb23e5.png)
.